crates/ty_python_semantic/src/python_platform.rs

```diff
@@ -16,7 +16,6 @@ pub enum PythonPlatform {
     /// We use a string (instead of individual enum variants), as the set of possible platforms
     /// may change over time. See <https://docs.python.org/3/library/sys.html#sys.platform> for
     /// some known platform identifiers.
-    #[cfg_attr(feature = "serde", serde(untagged))]
```

Comment:
`PythonPlatform` is used in the user-facing options struct: I don't see any schema changes, but we can't remove it here if it does change the schema (or we have to use another type in the user-facing options)

Reply:
I rewrote the serde implementations in a way that bincode accepts, but there should be no change in behavior.